resources/assets/js/components/JobBuilder/Intro/IntroForm.tsx

```diff
@@ -343,6 +361,7 @@ const IntroForm: React.FunctionComponent<IntroFormProps &
                     grid="base(1of1)"
                     component={SelectInput}
                     required
+                    disabled
```

Comment:
Should we just replace this with a `<p>` or similar? Are we looking to make it editable for different roles in the future?

Reply:
Yea we definitely could. @joshdrink what do you think? I wasn't sure what the final product should look like.

Reply:
Just talked to Josh. Next steps: 
- Remove select box and just use `<p>` tag
- Instead link to https://talent.test/en/settings#government in message